contracts/PetPark.sol

```diff
@@ -1,7 +1,105 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    enum AnimalType {
+        NotValid,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+    enum Gender {
+        Male,
+        Female
+    }
+
+    struct RenterInfo {
+        uint8 age;
+        Gender gender;
+    }
+    mapping(AnimalType => uint) public petsInShelter;
+    mapping(address => RenterInfo) public renters;
+    mapping(address => AnimalType) public rentedPet;
+    address public owner;
+
+    event Added(uint8 animalType, uint8 animalCount);
+    event Borrowed(uint8 animalType);
+    event Returned(uint8 animalType);
+
+    constructor() {
+        owner = msg.sender;
+    }
+
+    function add(uint8 animalType, uint8 count) external {
+        require(msg.sender == owner, "Not an owner");
```

Comment:
Although it is a design choice but better to use an enum here since it make the code more readable